src/app/components/auth-callback/auth-callback.component.css

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Is this file necessary since it's empty and there is a .scss?

Reply:
I think it came from a SCSS plugin in my IDE. I removed it